packages/@sanity/dashboard/package.json

```
@@ -20,6 +20,8 @@
   ],
   "dependencies": {
     "@sanity/image-url": "^0.140.19",
+    "@sanity/ui":"0.34.4",
+    "@sanity/icons":"1.1.2",
```

This may seem a little nitpicky, but tooling (npm, lerna) will sort these dependencies alphabetically and add spaces, so might as well do it consistently right away :)

```suggestion
    "@sanity/icons": "1.1.2",
    "@sanity/image-url": "^0.140.19",
    "@sanity/ui": "0.34.4",
```

---

Fixed :)